dockerfiles/README.md

```diff
@@ -16,3 +16,8 @@ By default, the image runs the Obscuro enclave service in SGX simulation mode. T
 non-simulation mode instead, run:
 
     docker run -e OE_SIMULATION=0 --privileged -v /dev/sgx:/dev/sgx -p XXX:11000/tcp obscuro_enclave --nodeID YYY --address :11000
+
+
+Stop and remove all obscuro docker containers:
```

Comment:
@otherview Capital 'O' in Obscuro for consistency.

Reply:
Done 👍